_posts/2021-01-25-solidity-developer-survey-2020-results.md

```diff
@@ -0,0 +1,359 @@
+title: 'Solidity Developer Survey 2020 Results'
+date: '2021-01-22'
+author: Franziska Heintel
+category: Announcements
+---
+
+Before we dive into the results we want to extend a big thank you to all of the Solidity developers that participated in the very first Solidity Developer Survey, which we [conducted at the end of last year](https://blog.soliditylang.org/2020/12/09/solidity-developer-survey-2020/)! 
+
+We were overwhelmed by the high quality of the submissions and are happy to extract important insights from your input.
+
+In this post, we'll be summarizing and commenting on the results of the survey. 
+
+Please note that none of the questions in the survey were mandatory. Hence, the amount of replies may vary per question.
+
+### Summary & Notable Insights
+
+- **Survey Audience**: 193 developers from all over the world replied to this survey, calling 48 different countries their current home. With roughly 20%, the big majority of respondents live in the United States of America, followed by India (8%) and Germany (7%). The majority of respondents speaks English at work.
+- **Developer Profiles**: Roughly 65% of respondents have been using Solidity for more than a year, with 43% being "Solidity pros" with 2 or more years of experience with the language. Looking at the overall coding seniority of the respondents it appears as if rather experienced developers start learning Solidity as opposed to code newbies starting to learn Solidity from scratch.
+- **Solidity Experience**: 43% of respondents rate themselves Solidity experts with an expertise level of 8 (out of 10) or higher.
+- **Solidity Developer Experience**: While noting many things that could still improve, there was an overall positive/good sentiment towards Solidity. Most respondents believed that the Solidity developer experience had some or much improvement over the last year.
+- **Solidity's Challenges**: The biggest challenge or problem in developer experience for most respondents is the availability and quality of tooling. Also seen as critical aspects in order to foster adoption are error handling, IDE integrations and better documentation.
+- **Most Liked, Dreaded and Anticipated**: Topics with room for imporovement that came up a lot: Error handling, debugging, the need for more tools and libraries, breaking dependencies.
+- **Contributing and Language Design**: There was surprisingly little interest and engagement in language design topics, partly because developers didn't know how to get involved, partly because they were not interested. (We will work on this! Efforts to foster collaborative language design are under way, we'll share more soon.)
+- **Community**: Almost 17% of participants in this survey do not interact with other Solidity developers at all!
+
+### (1) Survey Audience and Developer Profiles
+
+Firstly, let's have a look at the developers that participated in the survey. In total, we received **193 responses from 48 different countries**. 
```

Comment:
`-ly`?

Reply:
TIL [Some dictionaries don't even recognize `firstly` as a word](https://www.grammarly.com/blog/firstly/). I'm using it all the time :)